taskcluster/darwin-opt-base.tyml

```diff
@@ -42,7 +42,7 @@ payload:
       - >
         export TASKCLUSTER_ARTIFACTS="$(pwd)/public/" &&
         export TASKCLUSTER_ORIG_TASKDIR="$(pwd)" &&
-        (rm -fr ../tc-workdir/ ; mkdir ../tc-workdir/) && cd ../tc-workdir/ &&
+        (ls -lh .. && rm -fr ../tc-workdir && mkdir ../tc-workdir && ls -lh ..) && cd ../tc-workdir &&
```

Comment:
this might trigger advserse effects in the future, I remember fighting weird bugs because of `/` missing :)

Reply:
Hmm. This was causing problems because tc-workdir wasn't actually being deleted. So the build would pick up older files and fail. This change fixed that problem reliably but I don't know if there are other side effects hiding in the shadows.